version/build.rs

```diff
@@ -0,0 +1,12 @@
+use std::process::Command;
+
+fn main() {
+    if let Ok(git_output) = Command::new("git").args(["rev-parse", "HEAD"]).output() {
+        if git_output.status.success() {
+            if let Ok(git_commit_hash) = String::from_utf8(git_output.stdout) {
+                let trimmed_hash = git_commit_hash.trim().to_string();
+                println!("cargo:rustc-env=GIT_COMMIT_HASH={}", trimmed_hash);
```

Comment:
we should probably leave the `GIT_` prefix to git tools. let's call this `AGAVE_GIT_COMMIT_HASH` instead